examples/BasicSerialLogger/BasicSerialLogger.ino

```diff
@@ -0,0 +1,233 @@
+/**
+ * See documentation at https://nRF24.github.io/RF24Log
+ * See License information at root directory of this library
+ * Author: Witold Markowski (wmarkow)
+ */
+
+#include <Arduino.h>
+#include <string.h>
+
+#include <RF24Logger.h>
+#include <RF24StreamLogAppender.h>
+
+// Create hardware serial port log appender
+RF24StreamLogAppender rf24SerialLogAppender(&Serial);
+
+// Define global vendor id (it is stored in flash memory)
+const char vendorID[] PROGMEM = "RF24LogExample";
+
+
+// Define some test messages stored in EEPROM
+const char globalProgmemText[] PROGMEM

```

Comment:
just for future reference: doxygen has a problems when the `PROGMEM` identifier is placed after the variable name. Not important right now because this is example code (really I just want something upstream to play with), and doxygen won't need to interlink these c-strings with API docs.

According to the Arduino docs, this placement is only necessary if you're using a version of GCC that shipped with the Arduino IDE prior to v1.0

Reply:
I have written wrong in the code. It should be FLASH instead of EEPROM - that may be confusing.

>According to the Arduino docs, this placement is only necessary if you're using a version of GCC that shipped with the Arduino IDE prior to v1.0

Could you please give the link to that docs? I think the PROGMEM keyword is used by the developer in purpose if he wants to store a bigger portion of data (like strings for example) in FLASH memory, to save RAM.

Reply:
[link to arduino.cc docs](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwi9sra09NzvAhXTr54KHby-BwAQFjAAegQIBBAD&url=https%3A%2F%2Fwww.arduino.cc%2Freference%2Fen%2Flanguage%2Fvariables%2Futilities%2Fprogmem%2F&usg=AOvVaw2nO5ozF83EkEucCEfWL4kM) you may have to track down the exact version of GCC it refers to.

I see [the Arduino wiki docs](https://www.arduino.cc/en/pmwiki.php?n=Reference/PROGMEM) don't mention this

Reply:
>when the PROGMEM identifier is placed after the variable name

Now I see. PROGMEM keyword may be also put before the variable name but this may not work correctly in some older versions of Arduino IDE / GCC compiler.
Which is better for Doxygen?
```const PROGMEM dataType variableName[] = {};```
or
```const dataType PROGMEM variableName[] = {};```
?

Reply:
either will do; as long as `PROGMEM` does not come after the variable name

Reply:
Already done.